webrtc.html

```diff
@@ -1526,83 +1526,99 @@
+                          <li>
+                            <p> If no suitable transceiver is found
+                            (<var>transceiver</var> is unset), run the following
+                            steps:</p>
+                            <ol>
+                              <li>
+                                <p><a>Create an RTCRtpSender</a>, <var>sender</var>,
+                                from the media description.</p>
+                              </li>
+                              <li>
+                                <p><a>Create an RTCRtpReceiver</a>,
+                                <var>receiver</var>, from the media description.</p>
+                              </li>
+                              <li>
+                                <p><a>Create an RTCRtpTransceiver</a> with
+                                <var>sender</var>, <var>receiver</var> and
+                                <var>direction</var> set to <code>"recvonly"</code>,
+                                and let <var>transceiver</var> be the result.</p>
+                              </li>
+                            </ol>
+                          </li>
+                          <li>
+                            <p>Set <var>transceiver</var>'s <code><a data-for=
+                            "RTCRtpTransceiver">mid</a></code> value to the mid of
+                            the corresponding media description. If the media
+                            description has no MID, and <var>transceiver</var>'s
+                            <code><a data-for="RTCRtpTransceiver">mid</a></code>
+                            is unset, generate a random value as
+                            described in <span data-jsep=
+                            "applying-a-remote-desc">[[!JSEP]]</span>.</p>
+                          </li>
+                          <li>
+                            <p>If <var>direction</var> is <code>sendrecv</code> or
+                            <code>recvonly</code>, and
+                            <var>transceiver</var>'s
+                            <a>[[\CurrentDirection]]</a> slot
+                            is neither "sendrecv" nor "recvonly",
+                            <a>process the addition of a remote track</a> for
+                            the media description, given <var>transceiver</var>
+                            and <var>trackEvents</var>.</p>
+                          </li>
+                          <li>
+                            <p>If <var>direction</var> is
+                            <code>sendonly</code> or <code>inactive</code>, and
+                            <var>transceiver</var>'s
+                            <a>[[\CurrentDirection]]</a> slot
+                            is neither "sendonly" nor "inactive",
+                            <a>process the removal of a remote track</a> for
+                            the media description, given <var>transceiver</var>.</p>
```

Comment:
Here or in a separate issue/PR, do we want to do something similar for the removal of tracks and updating their muted state (which fires mute on track)? Currently it still says to "_Queue a task_, that if track.muted is false, will update the muted state of track with the value true." I suspect it should be "Add the track to tracksMuted" and before/after "for each trackEvents" we do "for each tracksMuted, if track.muted is false, update the muted state to true".